charts/nri-kube-events/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: A Helm chart to deploy the New Relic Kube Events
 name: nri-kube-events
-version: 1.5.0
-appVersion: 1.3.0
```

Comment:
I am not 100% sure that the CI is capable to bump both dependency and bundle in one PR.
We can either check or split those in two